libs/url/builder.js

```diff
@@ -65,7 +65,9 @@ module.exports.buildURL = function(opts) {
             urlObject.pathname = path.join(
                                     [TRANSFORMATION_PARAMETER, transformationString].join(transformationUtils.getChainTransformDelimiter()),
                                     urlObject.pathname
-                                )
+                                // Fix for Windows pathnames having backslashes instead of forward slashes. 
+                                // TODO: Replace path.join with a different library.
+                                ).replace("\\", "/")
```

Comment:
Instead of this, consider using `path.posix.join`, for example - 
```
urlObject.pathname = path.posix.join(
                                    [TRANSFORMATION_PARAMETER, transformationString].join(transformationUtils.getChainTransformDelimiter()),
                                    urlObject.pathname
                                )
```

Docs - https://nodejs.org/api/path.html#path_windows_vs_posix